starlight.sh

```diff
@@ -1,24 +1,27 @@
 package: STARlight
 version: "%(tag_basename)s"
-tag: r313-c
+tag: main
```

Comment:
I just noticed. Is this a branch? Please use a tagged version.

Reply:
In the JIRA discussion the main developers of STARlight were saying that they "moved" from the release of tagged versions to simply updating the main branch of the software. I do agree that tagged versions would be a better solution. Should I propose it to them? They were quite open in satisfying our request and the fact that the main branch compiles is already a good starting point imho.

Reply:
Just pick the hash of the current commit at the tip of the branch and use that. There is also `version` field you can use to give it a mnemonic name (e.g. the date of today). If we use the branch, there might be unexpected rebuilds, which we do not want, especially if the plan is to include this in O2sim / O2PDPSuite.

Reply:
Done.

Reply:
Thanks.